src/main/java/Ui/Ui.java

```diff
@@ -3,42 +3,65 @@
 import java.util.List;
 import Task.*;
 public class Ui {
-    public void uiGreet() {
+    public String uiGreet() {
+        String s = "Hello! I'm Daniel\nWhat can I do for you?\n";
         System.out.println("Hello! I'm Daniel\nWhat can I do for you?\n");
+        return s;
+
     }
-    public void uiBye() {
+    public String uiBye() {
+        String s = "Bye hope to see you again soon";
         System.out.println("Bye hope to see you again soon");
+        return s;
     }
-    public static void uiList(List<Task> array) {
```

Comment:
Perhaps it would be better to give a more meaningful name such as list instead of array?